src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1256,102 +1258,103 @@
+  "advancements.minecolonies.root.description": "Craft a Supply Camp or Ship and place it via right-clicking it on the ground",
   "advancements.minecolonies.place_townhall.title": "Start Your Colony",
-  "advancements.minecolonies.place_townhall.description": "Now's the time to start your colony. Grab that build tool and Town Hall block and find a nice, clear area. Then right-click with the build tool to select the style you want, where you want it, and other nifty things!",
-  "advancements.minecolonies.create_build_request_1_builder.title": "Start your Builder",
-  "advancements.minecolonies.create_build_request_1_builder.description": "Great, you've started your colony! To start building your colony up, you need to craft and place a Builder's Hut (use the build tool again), hire a Builder (automatic by default), and then tell them to build their hut!",
-  "advancements.minecolonies.click_gui_button_fulfill.title": "Fulfill a Request",
-  "advancements.minecolonies.click_gui_button_fulfill.description": "Now that you've told your Builder what to do, check out their Requests tab to see what resources or tools they need and fulfill one by clicking the fulfill button!",
-  "advancements.minecolonies.citizen_eat_food_first.title": "Feed your Citizens",
-  "advancements.minecolonies.citizen_eat_food_first.description": "Citizens, much like you, need to eat to be able to work. Put some food in your citizens' inventories and they'll eat it when they need to!",
-  "advancements.minecolonies.building_add_recipe_torch.title": "Teach Torch Crafting",
-  "advancements.minecolonies.building_add_recipe_torch.description": "You're probably going to get sick of having to craft everything for your workers. You can teach your Builder 2x2 recipes, go ahead and teach them how to craft a torch!",
-  "advancements.minecolonies.complete_build_request_1_builder.title": "Build the Builder's Hut",
-  "advancements.minecolonies.complete_build_request_1_builder.description": "If you keep your Builder fed and fulfill their requests, they should start and finish building your first hut!",
+  "advancements.minecolonies.place_townhall.description": "Place the Town Hall using the Build Tool - with this tool you can choose the style and hall position.",
+  "advancements.minecolonies.create_build_request_1_builder.title": "The Builder Begins",
+  "advancements.minecolonies.create_build_request_1_builder.description": "Craft and place a Builder's Hut with the Build Tool, hire a Builder (automatic by default), and tell them to build their hut!",
+  "advancements.minecolonies.click_gui_button_fulfill.title": "Fulfillment Comes Cheap",
+  "advancements.minecolonies.click_gui_button_fulfill.description": "Fulfil one of the Builder's requests for tools or resources using the buttons in their request screen",
```

Comment:
```suggestion
  "advancements.minecolonies.click_gui_button_fulfill.description": "Fulfill one of the Builder's requests for tools or resources using the buttons in their request screen.",
```

Reply:
Good catch. Though if you look at vanilla's advancements, they only use full stops when there's more than one sentence.